docs/en/observability/logs-parse-filter.asciidoc

```diff
@@ -0,0 +1,1227 @@
+      "host.ip": {
+        "gte": "192.168.1.100",<1>
+        "lte": "192.168.1.102"<2>
+      }
+    }
+  }
+}
+----
+<1> Greater than or equal to `192.168.1.100`.
+<2> Less than or equal to `192.168.1.102`.
+
+You'll get the following results only showing logs in the range you've set:
+
+[source,JSON]
+----
+{
+  ...
+  },
+  "hits": {
+    ...
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "ak4oAIoBl7fe5ItIixuB",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.101"
+          },
+          "@timestamp": "2023-08-08T13:45:12.123Z",
+          "message": "Disk usage exceeds 90%.",
+          "log": {
+            "level": "WARN"
+          }
+        }
+      },
+      {
+        "_index": ".ds-logs-example-default-2023.08.16-000001",
+        "_id": "bU4oAIoBl7fe5ItIixuC",
+        "_score": 1,
+        "_source": {
+          "host": {
+            "ip": "192.168.1.102"
+          },
+          "@timestamp": "2023-08-08T13:45:16.005Z",
+          "message": "User changed profile picture.",
+          "log": {
+            "level": "INFO"
+          }
+        }
+      }
+    ]
+  }
+}
+----
+
+[discrete]
+[[logs-stream-ip-ignore-malformed]]
+=== Ignore malformed IP addresses
+
+When you're ingesting a large batch of log data, a single malformed IP address can cause the entire batch to fail. Prevent this by setting `ignore_malformed` to `true` for the `host.ip` field. Update the `host.ip` field to ignore malformed IPs using the {ref}/indices-put-mapping.html[update mapping API]:
+
+[source,console]
+----
+PUT /logs-example-default/_mapping
+{
+  "properties": {
+    "host.ip": {
+      "type": "ip",
+      "ignore_malformed": true
+    }
+  }
+}
+----
+
+[discrete]
+[[logs-stream-reroute]]
+= Reroute log data to specific data stream
+
+preview::[]
+
+By default, an ingest pipeline sends your log data to a single data stream. To simplify log data management, use a {ref}/reroute-processor.html[reroute processor] to route data from the generic data stream to a target data stream. For example, you might want to send high-severity logs to a specific data stream to help with categorization. 
+
+This section shows you how to use a reroute processor to send the high-severity logs (`WARN` or `ERROR`) from the following example logs to a specific data stream and keep the regular logs (`DEBUG` and `INFO`) in the default data stream:
+
+[source,log]
+----
+2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%.
+2023-08-08T13:45:14.003Z ERROR 192.168.1.103 Database connection failed.
+2023-08-08T13:45:15.004Z DEBUG 192.168.1.104 Debugging connection issue.
+2023-08-08T13:45:16.005Z INFO 192.168.1.102 User changed profile picture.
+----
+
+NOTE: When routing data to different data streams, we recommend keeping the number of data streams relatively low to avoid oversharding. See {ref}/size-your-shards.html[Size your shards] for more information.
```

Comment:
This warning might be overkill at this stage. The user just got started with routing and already has to think about shard sizes etc. Lets have this rather on the doc side for the reroute processor.

Reply:
I believe I added this at @felixbarny request, so I'm not sure if he has a strong opinion one way or another, but I see your point and am open to removing it.

Reply:
@felixbarny WDYT? I'm hoping at this stage of the users journey they don't have to think about this yet and only worry later about it.

Reply:
I think there should be some mentioning of that early on. If we don't, there's a risk that users may route on a high cardinality field which then leads to frustration later on.

I don't think that users need to read the full shard sizing guide at this point. But they should know that they shouldn't overdo it.

Reply:
The part that sticks out to me is `oversharding`. We expose the user to a term (shard) which I would hope a user should never have to think about especially serverless. Maybe we can rephrase it to leave out the shard part but still link to it. Something like:

> NOTE: When routing data to different data streams, we recommend picking a field with low cardinality to prevent explosion of the number of data streams. For more details see {ref}/size-your-shards.html[Size your shards].

I don't like where the link points, much rather also in the target users would think about number of data streams not shards.